### ISSUE TITLE
Text is illegible on a 5k screen

### ISSUE THREAD

**Issue by jackmcdade** [jdan/98.css#56]
It's so razor thin I can't really read it anymore. 
![image](https://user-images.githubusercontent.com/44739/80256731-4cf5c380-864d-11ea-87e4-2d13491369a4.png)

**Comment by brantwedel** [comment]
Maybe you can try setting a scaled resolution of 640x480 ;-)

**Comment by alete89** [comment]
1st world problems

**Comment by jackmcdade** [comment]
@alete89 Just saying, it was fine before and now it's not. I don't _need_ to use it, just was being helpful.

**Comment by jdan** [comment]
@jackmcdade wanna give me a full-size screenshot? Unsure what a fix might look like but def wanna look into it

**Comment by jackmcdade** [comment]
Sure thing! I don't know if there's much to be done, or if it will even be obvious how hard it is to read if you're not on a 5k monitor, but here you go. FYI, I'm using the LG 5k for Macs.


![jdan github io_98 css_](https://user-images.githubusercontent.com/44739/80389292-e3f29380-8878-11ea-9b15-9b8171c79140.png)

**Comment by jdan** [comment]
Unfortunately the screenshot doesn't do it justice. Will wait to see if any onlookers have this monitor and may know of a fix.

**Comment by bergamin** [comment]
You may just need to set your monitor to 800x600 :rofl:

**Comment by padraigfl** [comment]
Isn't this just the difference between the width of a pixel on a 15" 640x480 Packard Bell (52PPI) vs a 27" 5120x2880 iMac (217PPI)

Possible fixes:
- marvel at how far we've come
- transform scale at top level if over a certain screen width (think I done this with one thing, everything else needs to be aware of it though)
- _maybe_ just double the size of *everything* if the window width is in excess of 2K

**Comment by piranna** [comment]
Probably there's a way to get and use actual pixel density, similar to how 2x images are being downloaded but for all the page content? And if so, does we want anti-alised text, or razor edges?

**Comment by Zlitus** [comment]
Even on a MacBook Pro in pixelRatio *2, some texts are very difficult to read (Especially the labels of the dropdown menus and the <details> bold texts). Don't know if related.

**Comment by nszceta** [comment]
HiDPI scaling is a persistent Windows platform issue that remains unresolved for older programs to this day.

> Today, many existing programs have been updated with APIs that allow them to properly scale their interface designs to maintain usability on high-DPI displays. Microsoft didn’t implement anything like that on Windows. That means if the app or program doesn’t know how to scale for high-DPI displays, then you’re out of luck and it will display pixel for pixel. With a 300 dpi pixel density, those UI designs made for 72 pixel per inch displays would shrink so that every normal inch of that interface would now fit into about a quarter of an inch on your screen.

[Source](https://pocketnow.com/hidpi-scalling-in-windows-10)

**Comment by Zlitus** [comment]
@nszceta : Yes I understand very well this, but it is completely unrelated. What you quote is a text explaining than apps have to take care of HighDPI to adapt their interface on Windows;
And what we say is than there is a readability problem with the font in some case with 98.css.
